content/chapter-03/Intro.md

```diff
@@ -3,11 +3,11 @@
 * [Application metrics](./ApplicationMetrics.md)
   * [Linux System Metrics](./LinuxSystemMetrics.md)
   * [Windows System Metrics](./WindowsSystemMetrics.md)
-* Lab: common tools for application metrics
+* [Lab: common tools for system metrics](./SystemMetricsLab.md)
```

Comment:
Lab: Tools for Measuring Resource Utilization

Reply:
Done